### PR TITLE
Provision references citator handles French

### PR DESCRIPTION
**Pull request by longhotsummer**
* make provision refs grammar choose different parse trees based on the current language; this means that only one language is used when parsing, so languages can have terms that clash
* add basics of French provision lookups

### CODE REVIEW

File: indigo/tests/test_provision_refs.py [hunk 1]
```diff
@@ -1770,15 +1825,32 @@ def test_target_truncated(self):
...
 
-        result = parse_provision_refs("Afdeling 2 daarvan")
+        result = parse_provision_refs("Afdeling 2 daarvan", "afr")
         self.assertEqual("thereof", result.target)
         self.assertEqual(result.end, 18)
 
-        result = parse_provision_refs("Afdeling 2 van die Wet met kak")
-        self.assertEqual("of", result.target)
-        self.assertEqual(result.end, 15)
+        result = parse_provision_refs("Afdeling 2 van die Wet met kak", "afr")
```
Comment (goose-life) [inline indigo/tests/test_provision_refs.py:1837]: 😂

**Review by goose-life — APPROVED** [review]

File: indigo/tests/test_provision_refs.py (hunk 1 above)
Comment (longhotsummer) [reply indigo/tests/test_provision_refs.py]: translation courtesy of copilot 🤣

**Comment by longhotsummer** [comment]
i think this is good enough to ship and start testing with actual data; FRA will have more edge cases, but this will handle basics

**Review by goose-life — APPROVED** [review]
